File: WEEK 1/Python/Calculate Area of a Circle .py
import math

def calculate_area(radius):
    return math.pi * radius**2

while True:
    try:
        radius = float(input("Enter the radius of the circle (non-null positive value): "))

        if radius <= 0:
            print("Please enter a non-null positive value for the radius.")
        else:
            area = calculate_area(radius)
            print(f"The area of the circle with radius {radius} is {area:.2f}")
            break 
    except ValueError:
        print("Invalid input. Please enter a valid number for the radius.")

File: WEEK 1/java/Calculate Area of a Circle.java
import java.util.Scanner;

public class CircleAreaCalculator {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        // Define the constant value of π
        final double PI = 3.141593;

        System.out.print("Enter the radius of the circle: ");
        double radius = input.nextDouble();

        // Calculate the area of the circle using the formula
        double area = PI * Math.pow(radius, 2);

        System.out.println("The area of the circle with radius " + radius + " is: " + area);

        input.close();
    }
}
